### PR TITLE
Add policy for www.haskell.org repo access

### PR DESCRIPTION
**Pull request by tomjaguarpaw**
Draft policy tackling: https://github.com/haskell-org/committee/issues/9

Committee: Hopefully we can vote on this in committee meeting on Fri 18th Feb 2022.

### CODE REVIEW

File: proposals/0004-haskell-org-repo-maintainers [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+and source repositories associated with the www.haskell.org website.
+The settings and source repositories are currently hosted on the
+GitHub platform, at
+<https://github.com/haskell-infra/www.haskell.org/>.
+
+* All members of the Haskell.org committee should have write access,
+  to be used for carrying out their normal duties around the website.
+
+* All members of the Haskell.org committee will, on request, be
+  granted the highest level of admin access.
```
Comment (jaspervdj) [inline proposals/0004-haskell-org-repo-maintainers:19]: We should have at least 2/3 admins for redundancy(?)

File: proposals/0004-haskell-org-repo-maintainers [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+discussion: https://github.com/haskell-org/committee/pull/11
+---
+
+# Write access to haskell.org website repo
+
+This policy determines who should have write access to any settings
+and source repositories associated with the www.haskell.org website.
+The settings and source repositories are currently hosted on the
+GitHub platform, at
+<https://github.com/haskell-infra/www.haskell.org/>.
```
Comment (jaspervdj) [inline proposals/0004-haskell-org-repo-maintainers:13]: Maybe add a paragraph about even though we're limiting write access, we're happy to accept PRs etc.

**Review by jaspervdj — COMMENTED** [review]
Two nitpics

**Comment by tomjaguarpaw** [comment]
Thanks @jaspervdj, I've incorporated both of your suggested changes.

Hopefully we can vote on this in the committee meeting on Wed 30th Mar 2022.

**Review by rebeccaskinner — APPROVED** [review]
This looks good to me, and we have voted to approve this in the call on 2022-03-30

**Review by ryantrinkle — APPROVED** [review]

**Pull request merged**